### PR TITLE
Add Turkish translation

### PR DESCRIPTION
**Pull request by molcay**
Related to #106.

### CODE REVIEW

File: i18n/tr.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
+{
```
Comment (martinheidegger) [inline i18n/tr.json:1]: Is this still work in progress? There are many texts still in english.

Comment (molcay) [reply i18n/tr.json:1]: oh, I missed this file portion. I will be add the missing translation, ASAP.

Comment (molcay) [reply i18n/tr.json:1]: I added  the missing translated files.

**Comment by martinheidegger** [comment]
Published as 6.1.0

**Pull request merged**